concepts/basics/introduction.md

```diff
@@ -1,30 +1,30 @@
 # Introduction
 
-Java is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable is defined by explicitly specifying its type.
+Java is a statically-typed language, which means that the type of a variable is known at compile-time. Assigning a value to a name is referred to as defining a variable. A variable is defined by explicitly specifying its type.
 
 ```java
 int explicitVar = 10;
 ```
 
-Updating a variable's value is done through the `=` operator. Once defined, a variable's type can never change.
+Updating a variable's value is done through the `=` operator. Here, `=` does not represent mathematical equality. It simply assigns a value to a variable. Once defined, a variable's type can never change.
 
 ```java
 int count = 1; // Assign initial value
 count = 2;     // Update to new value
 
-// Compiler error when assigning different type
+// Compiler error when assigning a different type
 // count = false;
 ```
 
-Java is an [object-oriented language][object-oriented-programming] and requires all functions to be defined in a _class_. The `class` keyword is used to define a class.
+Java is an [object-oriented language][object-oriented programming] and requires all functions to be defined in a _class_. The `class` keyword is used to define a class.
```

Comment:
This change should be reverted, as `[object-oriented-programming]` is a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links),  and refers to the link defined at the bottom of the file.

Reply:
I think that's great 👍🏼

Reply:
OK so if you could push a change that reverts the change to the Markdown link I'll be happy to approve it for you!

Reply:
Okay. I have edited it. Let me know if everything good to go now.